rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -197,7 +197,7 @@ components:
           type: string
           format: date-time
           description: The server time when the error was generated.
-          example: 2019-05-09T14:47:13.101+0000
+          example: 2019-05-09T14:47:13.101+00:00
```

Comment:
This appears as `example: OrderedMap {}` Swagger:
![image](https://user-images.githubusercontent.com/8041246/66509526-1af18b00-ea88-11e9-96b9-ce23243cc5fc.png)
I would expect to see the same value.

Reply:
I had the ingest service fail to read the date while testing because the time without the colon is malformed.

Reply:
Adding quotes fixes the issue in Swagger:
```yaml
example: '2019-05-09T14:47:13.101+00:00'
```
![image](https://user-images.githubusercontent.com/8041246/66515188-770ddc80-ea93-11e9-9bf9-0e64b8b77db3.png)